pkg/api/client/client.go

```diff
@@ -0,0 +1,111 @@
+package client
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/bmc-toolbox/common"
+	rivets "github.com/metal-toolbox/rivets/types"
+)
+
+const (
```

Comment:
I like the idea of a client. But instead of copying the values of the endpoints, we should export them from the `routes` module instead. I suggest gathering all of those strings into a file called `endpoints.go` or something and making sure to import them here. That will protect us from route changes in the future (like removing the `api` token from the endpoint).